### DISCUSSION TITLE
Eintrag für eine KIM-Testadresse

### DISCUSSION THREAD

**Discussion by omeiners** [gematik/spec-E-Rezept-ServiceRequest#77] [Share]
### Hersteller (Manufacturer)

NARZ-AVN easytax

### Kontaktmöglichkeit (Contact Option)

Alexander Goldbeck und Ommo Meiners
Rezeptanforderung-Test@narz-avn.de

### Art des PS (Type of PS)

AVS / Taxierungssoftware

### KIM-Adresse

easytax@ibm.kim.telematik-test

### Unterstützte Anwendungsfälle (Supported Use Cases) aus [AppTagsCS](https://simplifier.net/app-transport-framework/app-tags-cs)

- [X] atf
- [X] eRezept_ParenteraleZubereitung
- [X] eRezept_Rezeptanforderung
- [X] eRezept_Zuweisung
- [ ] eRezept_Kommunikation

**Comment by florianschoffke** [comment]
Vielen Dank, habe ich in https://github.com/gematik/spec-E-Rezept-ServiceRequest/discussions/61 aufgenommen
